### PR TITLE
HR and BP enhancement

### PR DESCRIPTION
**Pull request by gienkov**
* It will display 0 HR and fail to find BP on limbs that have TQ on.
* Check for TQ is run only if medic is checking pulse on arms.

### CODE REVIEW

File: addons/medical/functions/fnc_actionCheckPulseLocal.sqf [hunk 1]
```diff
@@ -46,9 +46,14 @@ if (_heartRate > 1.0) then {
     };
 };
 
+if (_selectionName in ["hand_l","hand_r"] && [_unit, _selectionName] call FUNC(hasTourniquetAppliedTo)) then {
```
Comment (jokoho48) [inline addons/medical/functions/fnc_actionCheckPulseLocal.sqf]: add here lazyEelevation that could Improve the Performance
``if (_selectionName in ["hand_l","hand_r"] && {[_unit, _selectionName] call FUNC(hasTourniquetAppliedTo)}) then {`

**Comment by Glowbal** [comment]
Doesn't take all limbs into account (`leg_l` and `leg_r`)

**Comment by gienkov** [comment]
You don't check BP/HR on legs that's why check for arms only.

**Comment by gienkov** [comment]
Did the same for BP, it will display ***You fail to find a blood pressure*** (it's in stringtable).

**Pull request merged**